<!DOCTYPE html>

<html>

    <head>
        <title>Generation of Covalently Closed Circular DNA of
            Hepatitis B Viruses via Intracellular Recycling Is
            Regulated in a Virus Specific Manner</title>
            <!-- this shows the figures which inspired my thesis on HBV infection kinetics-->

    </head>

    <body>

        <h1>Relevant data from "Generation of Covalently Closed Circular DNA of Hepatitis B Viruses via Intracellular Recycling Is Regulated in a Virus Specific Manner"</h1>
        <h3> Josef Kock, Christine Rosler, Jing-Jing Zhang, Hubert E. Blum, Michael Nassal, Christian Thoma</h3>
            <a href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1001082">Check it out online</a>
            <br>
            <a href="#figures">Figures</a>
            <h3>Abstract</h3>

            <p>Persistence of hepatitis B virus (HBV) infection requires covalently closed circular (ccc)DNA formation and amplification,
                which can occur via intracellular recycling of the viral polymerase-linked relaxed circular (rc) DNA genomes present in
                virions. Here we reveal a fundamental difference between HBV and the related duck hepatitis B virus (DHBV) in the recycling
                mechanism. Direct comparison of HBV and DHBV cccDNA amplification in cross-species transfection experiments showed
                that, in the same human cell background, DHBV but not HBV rcDNA converts efficiently into cccDNA. By characterizing the
                distinct forms of HBV and DHBV rcDNA accumulating in the cells we find that nuclear import, complete versus partial release
                from the capsid and complete versus partial removal of the covalently bound polymerase contribute to limiting HBV
                cccDNA formation; particularly, we identify genome region-selectively opened nuclear capsids as a putative novel HBV
                uncoating intermediate. However, the presence in the nucleus of around 40% of completely uncoated rcDNA that lacks
                most if not all of the covalently bound protein strongly suggests a major block further downstream that operates in the HBV
                but not DHBV recycling pathway. In summary, our results uncover an unexpected contribution of the virus to cccDNA
                formation that might help to better understand the persistence of HBV infection. Moreover, efficient DHBV cccDNA
                formation in human hepatoma cells should greatly facilitate experimental identification, and possibly inhibition, of the
                human cell factors involved in the process.</p>

                <h3 >Introduction</h3>
                <p>Currently, more than 350 million people suffer from chronic
                    HBV infection. Chronic hepatitis B frequently progresses to liver
                    cirrhosis and hepatocellular carcinoma, a leading cause of cancer-
                    related morbidity and mortality worldwide [1,2].
                    HBV is a small enveloped hepatotropic DNA virus which
                    replicates by reverse transcription of an RNA intermediate, the
                    pregenomic (pg) RNA (for review: [3,4]), to yield encapsidated,
                    partially double-stranded rcDNA to which the viral polymerase is
                    covalently bound [5]. Upon infection, rcDNA is transported to the
                    host cell nucleus where it is converted into cccDNA (Figure S1).
                    Episomal cccDNA then acts as template for all viral transcripts.
                    These include the subgenomic RNAs encoding the surface
                    proteins, and the pgRNA that serves as mRNA for the polymerase
                    protein and the capsid, or core, protein. Binding of polymerase to
                    the RNA stem-loop structure e initiates packaging of one pgRNA
                    molecule per newly forming capsid and its reverse transcription.
                    The first product is single-stranded (ss) DNA ofminus polarity; due
                    to the unique protein-priming mechanism, its 59 end is, and
                    remains, covalently linked to the polymerase. The pgRNA is
                    concomitantly degraded, except for its 59 terminal ,15–18
                    nucleotides which serve as primer for plus-strand DNA synthesis,resulting in rcDNA and, as a side-product, some double-stranded
                    linear (dl) DNA. DNA-containing capsids are then enveloped by
                    surface proteins and cellular lipids and secreted as virions.
                    Alternatively, they are redirected to the nucleus to increase
                    cccDNA copy number by a mechanism termed intracellular
                    recycling; many estimates for cccDNA copies per infected
                    hepatocyte are in the range of 5 to 50 [6]. This amplification
                    prevents loss during cell division of the cccDNA which can not be
                    replicated semiconservatively [7]. Thus, cccDNA formation and
                    recycling are central to establish and maintain persistent infection,
                    and they limit the efficacy of antiviral nucleot(s)ides in the
                    treatment of chronic hepatitis B, as these do not directly target
                    cccDNA (for review: [8]).</p>
                    <p>However, despite its central importance the molecular pathway
                    driving the conversion of HBV rcDNA to cccDNA is poorly
                    understood. In vivo studies face numerous challenges. Liver
                    biopsies from human patients are scarce, affected by the natural
                    history of infection, and they sample only a small volume of the
                    liver; recent estimates for cccDNA content in infected human liver
                    vary from 0.01 to 1.4 [9] to 0.1 to 10 copies per hepatocyte [10],
                    with large patient-to-patient variation. Kinetic studies during acute
                    infection, doable exclusively in chimpanzees, showed peak values
                    of around 10 cccDNA copies per infected cell but much lower
                    numbers before and after [11]. In well controllable experimental
                    settings, on the other hand, such as transfected cell lines [12–14] or
                    HBV-transgenic mice [15,16], HBV produces little if any
                    cccDNA.</p>
                    <p>DHBV is a related animal virus that is widely used as a model to
                    study HBV infection [17]. Like HBV, DHBV produces cccDNA
                    in cultured duck hepatocytes and in vivo [7,18], with reported
                    mean values of 2.9 to 8.6 copies per hepatocyte [19] though also
                    with temporal and cell-to-cell fluctuations (from one to .36 copies
                    per cell). Importantly, DHBV transfected into the chicken
                    hepatoma line LMH also generates well detectable amounts of
                    cccDNA. Initially in primary duck hepatocytes, then using that
                    system, Summers and colleagues had first shown that knocking out
                    surface protein expression, and thus virion secretion, dramatically
                    increased cccDNA copy numbers per nucleus to about 200–400
                    molecules [20,21]. Recent studies in stably or transiently
                    transfected human cell lines suggest that preventing HBV surface
                    protein expression also stimulates cccDNA formation, but to a
                    much lower degree; instead, a ‘‘protein-free’’ form of rcDNA (pf-
                    rcDNA) accumulated [12,13]. The term protein-free (which we
                    will adhere to here) was operationally defined by the partitioning
                    of this rcDNA form into the aqueous phase upon phenol
                    extraction without prior proteinase K (PK) treatment (which
                    artificially degrades the protein); polymerase-linked DNA parti-
                    tions into the organic phase. ‘‘Protein-free’’ does therefore not
                    imply the complete absence of any amino acid from the DNA.</p>
                    <p>Though not finally proven, several lines of indirect evidence
                    suggest that pf-rcDNA is a precursor to cccDNA [12,13]; not the
                    least, removal of the bound polymerase is a sine qua non for
                    cccDNA formation. Caveats are that nicked cccDNA, generated
                    to some extent during preparation and naturally protein-free, has
                    the same electrophoretic mobility as pf-rcDNA. Furthermore,
                    Southern blots from infected liver nuclei have generally shown
                    only little rcDNA versus cccDNA [22]. However, in such samples
                    one usually looks at an established pool of cccDNA whereas therecent cell culture studies monitored initial cccDNA formation.
                    Potential precursors accumulating under these conditions may not
                    be detectable anymore in in vivo samples.</p>
                    <p>At any rate, the initial presence of protein-bound rcDNA inside
                    virions and eventually ofnuclear cccDNA, associated with histones
                    [23], requires as intermediate steps nuclear transport of the
                    rcDNA genome, its release from the viral capsid and removal of
                    the bound polymerase to allow generation ofprecisely one genome
                    length equivalents of the plus- and the minus-strand before final
                    ligation into cccDNA (Figure S1B). The order of events is not
                    firmly established. Intact nucleocapsids may deliver the protein-
                    bound rcDNA to the nucleus where its release from the capsid and
                    polymerase removal are mediated by host factors; this view is
                    supported by the minimalistic genomes of hepadnaviruses (only
                    ,3 kb) and by data from nuclear transport model systems [24,25].
                    Alternatively, the nucleocapsid itself may contain corresponding
                    activities such that polymerase removal could precede capsid
                    release, possibly already in the cytoplasm. Some evidence in favor
                    of this view has recently been forwarded [13,26].</p>
                    <p>Apart from these mechanistic aspects it appears, in essence, that
                    DHBV in the avian LMH cells produces much more cccDNA
                    than HBV in human hepatoma cells. One conceivable explanation
                    are cell-specific differences. For instance, the routinely used
                    human HepG2 and Huh7 hepatoma cell lines may lack enzymatic
                    activities required for cccDNA formation that are present in the
                    avian cells. Alternatively, the different efficiencies in cccDNA
                    formation may be a feature of the respective viruses.</p>
                    <p>In order to address this question we took advantage of the
                    principal ability of hepadnaviruses to replicate in hepatoma cell
                    lines of heterologous species origin. After transfection HBV is
                    capable of producing rcDNA in LMH cells and the same holds for
                    DHBV in HepG2 and HuH7 cells [27,28]. However, cccDNA
                    formation in such cross-species transfections has not yet been
                    addressed. Here we performed such experiments and found that,
                    unexpectedly, the major contribution to cccDNA formation comes
                    from the virus rather than from the cell.</p>

                    <h3>Results</h3>
                    <h4>Intracellular recycling of HBV genomes occurs at very low
                        levels</h4>
                    <p>Detection of cccDNA by Southern blotting can severely be
                        hampered by the presence of ssDNA species which have a similar
                        electrophoretic mobility and are often present in excess. A further
                        problem in transient transfections is the highly abundant plasmid
                        DNA. We therefore developed an assay that essentially eliminates
                        ssDNA and plasmid DNA by double-digestion with the restriction
                        enzyme Dpn I and Plasmid safe DNase (PsD). Dpn I requires
                        bacterially methylated DNA to be active and selectively restricts
                        the transfected plasmid. PsD digests single-stranded (ss) and
                        double-stranded linear (dl) but not circular molecules such as
                        cccDNA. Although it has been surmised that rcDNA is a substrate
                        for PsD [9], our own preliminary data suggested this holds only for
                        very immature rcDNA forms. To enhance cccDNA production,
                        we used plasmids coding for surface-deficient HBV and DHBV.</p>
                        <p>Because cccDNA is enriched in the nucleus, we separated nuclei
                        from cytoplasm by treating the cells with the mild detergent NP-40
                        and subsequent centrifugation. To identify encapsidated DNAs,
                        we incubated the cytoplasmic extracts with micrococcal nuclease
                        (MN) which digests free nucleic acids but not those protected
                        inside capsids. Finally, because protein-bound DNA is neither
                        recovered upon phenol extraction nor using the silica column
                        adsorption (QIAamp) method employed here, all initial DNA
                        preparations included a PK treatment. The results for DHBV in LMH and HBV in HepG2 cells are
                        shown in Figure 1. Treatment ofthe cytoplasmic samples with MN
                        revealed the common replicative intermediates, i.e. rcDNA,
                        dlDNA and ssDNA. Digestion with Dpn I only produced a
                        similar pattern, except that additional plasmid-derived bands (Pla)
                        were visible. Nuclear DNA treated with Dpn I alone produced a
                        similar pattern, yet as expected, knock-out of surface protein
                        expression enhanced the cccDNA signal, particularly for DHBV.
                        Additional treatment of nuclear DNA with PsD removed all bands
                        except those at the rcDNA and cccDNA positions; the equal signal
                        intensities before and after PsD treatment demonstrated that
                        mature rcDNA was not appreciably attacked by PsD. For HBV, a
                        band at the cccDNA position was exclusively visible in the surface-
                        deficient background (Figure 1B). However, the nuclear rcDNA
                        signal was much more enhanced, in line with recent reports
                        [12,13]. Comparable results were obtained in Huh7 cells (Figure
                        S2A). Together, these data demonstrated that the applied
                        procedure enabled the reliable detection of cccDNA and rcDNA,
                        without interference from other virus- or plasmid-derived nucleicacids. Furthermore, they confirmed that DHBV in avian cells
                        produces much more cccDNA than HBV in human cells.</p>
                        <p>Faint bands with cccDNA-like mobility were also detectable in
                        the cytoplasmic fractions of those samples containing well visible
                        nuclear cccDNA; ethidium bromide staining of the agarose gel
                        used to generate the blot revealed indeed some chromosomal
                        DNA in the cytoplasmic samples, indicating an incomplete
                        separation of two fractions. This prompted us to employ a more
                        efficient separation procedure in later experiments (see below).</p>
                        <figure id="figures">
                            <img src="images/writingfig1.jpg" width="350" alt="A southern blot">
                            <figcaption><strong>Figure 1. DHBV replication in avian cells and HBV replication in
                                human cells.</strong> (A) Chicken LMH cells transfected with wild-type and
                                surface-deficient DHBV. (B) Human HepG2 cells transfected with wild-
                                type and surface-deficient HBV. DNAs were extracted, after prior PK
                                digestion, from cytoplasmic lysates and the nuclear fractions. One
                                aliquot of the cytoplasmic lysates was treated with micrococcal
                                nuclease (MN) before DNA extraction and analyzed without further
                                treatment. All other samples were incubated, post extraction, with
                                either Dpn I alone (Dpn), or Dpn I plus Plasmid safe DNAse (Dpn+PsD).
                                Nomenclature of the various DNA species: RC, relaxed circular; DL,
                                double strand linear; SS, single strand; CCC, covalently closed circular
                                DNA; Pla, Dpn I restriction fragments of transfected plasmid DNA.</figcaption>
                        </figure>

                        <h4>The virus, not the cell, is the major determinant for
                            efficient cccDNA formation</h4>
                            <p>Next we performed analogous experiments with DHBV
                                transfected into the human cell lines, and HBV transfected into
                                LMH cells (Figure 2). For DHBV, cytoplasmic extracts treated
                                with MN produced a similar pattern of replicative intermediates in
                                HepG2 (Figure 2A) and HuH7 cells (Figure S2B) as in LMH cells
                                (Figure 1A). HBV in LMH cells, compared to the human cell lines,
                                generated a more complex pattern with a distinct band of an
                                intermediate mobility between rcDNA and ssDNA (Figure 2B).
                                This additional band originates from strongly enhanced splicing of
                                the HBV pgRNA in the chicken cell line (Ko¨ck, Nassal, Thoma;
                                unpublished data). The spliced genomes are of linear conforma-
                                tion and therefore accessible to PsD digestion. Most importantly
                                for the current study, the envelope-deficient DHBV produced a
                                strong cccDNA signal in both human cell lines (Figure 2A and
                                Figure S2B), with an intensity equaling that of the nuclear rcDNA
                                as in LMH cells. Conversely, HBV in LMH cells generated a
                                similar pattern as in the human cell lines, with a relatively strong
                                band at the rcDNA yet only a weak band at the cccDNA position
                                (Figure 2B). The HBV specific lack of effect of the envelope knock-
                                out in LMH cells correlated with a much lower abundance of the
                                surface protein-coding mRNAs compared to the human cells (data
                                not shown). Thus wild-type HBV in LMH cells is phenotypically
                                similar to its envelope-deficient counterpart.</p>
                                <p>Together, these data demonstrated that both HepG2 and
                                HuH7 cells are competent to support cccDNA synthesis, yet with
                                strikingly higher efficiency for DHBV than HBV; conversely,
                                LMH cells did not support more efficient cccDNA formation for
                                HBV. These results were reproduced in about twenty independent
                                experiments. Furthermore, this virus-specific difference was
                                neither changed by increasing pgRNA levels via replacement of
                                the HBV core promoter by the CMV promoter, nor by analyzing
                                the cultures at earlier or later time points post transfection (Figure
                                S3). Thus the virus contributes more profoundly to the efficiency
                                of cccDNA biosynthesis than the cell. More quantitative data on
                                the amounts and intracellular distribution of the different viral
                                DNA were obtained with samples from gradient purified nuclei.</p>

                                <figure>
                                    <img src="images/writingfig2.jpg" alt="Another Southern blot" width="350">
                                    <figcaption><strong>Figure 2. DHBV replication in human cells and HBV replication
                                        in avian cells.</strong> (A) DHBV in HepG2 cells. (B) HBV in LMH cells. Labeling
                                        and abbreviations are as in the legend to Figure 1. The position of a
                                        major DNA product of about 2 kb in length derived from enhanced HBV
                                        splicing in LMH cells is indicated (spl).</figcaption>
                                </figure>



                        




    </body>




</html>